wp-theme-2018/common_menu.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+function call_menu_api_microservice($home_page_url, $url_site, $lang, $call_type)
+{
+    $main_post_page = get_option('page_for_posts');
+    if (! function_exists("pll_get_post")) {
+        # Menus and siblings require Polylang.
+        return [];
+    }
+    $current_language_page_id = pll_get_post($main_post_page, $lang);
+    $main_post_page_name = urlencode(get_the_title($current_language_page_id));
+    $main_post_page_url = get_permalink($current_language_page_id);
+
+    $urlApi = 'http://' . (getenv('MENU_API_HOST') ?? "menu-api") . ':3001/menus/'.$call_type.'/?lang=' . $lang . '&url=' . trailingslashit( $url_site ) .
```

Comment:
Please rename `$urlApi` to something without camelCase, e.g. `$api_url`.

Please use the new-style PHP 8 string expansion instead of a series of calls to the concatenation operator (`.`). If you must use `.`, please treat it as an operator i.e. one space before and after, so that casual readers won't confuse it with a method call or something.